extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -230,6 +230,7 @@ public function testGetContext(
 		$customDartKvs = 'a=b;c=d';
 		$catId = WikiFactoryHub::CATEGORY_ID_LIFESTYLE;
 		$shortCat = 'shortcat';
+		// @todo(mech); change this once $wgEnableLocalResourceLoaderLinks is enabled globally
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)